shell/assets/translations/en-us.yaml

```diff
@@ -1770,7 +1770,9 @@ cluster:
   providerGroup:
     create-custom1: Use existing nodes and create a cluster using RKE
     create-custom2: Use existing nodes and create a cluster using RKE2/K3s
-    create-kontainer: Create a cluster in a hosted Kubernetes provider
```

Comment:
If you are changing an existing line of code, comment out the original line and add the new line, both within the Start/End markers

Reply:
Ok, I'll make this change and let you know after I re-test it

Reply:
I made commit [7ca49d5](https://github.com/verrazzano/rancher-dashboard/pull/100/commits/7ca49d51d62be821eb5bc62f223c5c7bff75b77b) to address this. I'm waiting for a build to re-test it. Let me know if this is the change you are expecting